### PR TITLE
feat: Add statusBadge for extensibility

### PR DESCRIPTION
**Pull request by OliwiaGowor**
**Description**

Changes proposed in this pull request:

- added statusBadge for extensibility Status section
- adjusted documentation

**Related issue(s)**
#3070 

**Definition of done**

- [x] The PR's title starts with one of the following prefixes:
  - feat: A new feature
  - fix: A bug fix
  - docs: Documentation only changes
  - refactor: A code change that neither fixes a bug nor adds a feature
  - test: Adding tests
  - chore: Maintainance changes to the build process or auxiliary tools, libraries, workflows, etc.
- [x] Related issues are linked. To link internal trackers, use the issue IDs like `backlog#4567`
- [x] Explain clearly why you created the PR and what changes it introduces
- [x] All necessary steps are delivered, for example, tests, documentation, merging

### CODE REVIEW

File: docs/extensibility/30-details-summary.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ This table lists the available parameters of the **data.details.header**, **data
 
 Extra parameters might be available for specific widgets.
 
+#### Status Header
+
+The **data.details.status.header** accepts one widget (for example [Badge](./50-list-and-details-widgets.md#badge)), displayed on the right side of the **Status** section header. This widget is used to summarize the status of the resource.
+
+<img src="./assets/display-widgets/StatusBadge.png" alt="Example of a status badge widget" width="30%" style="border: 1px solid #D2D5D9">
```
Comment (OliwiaGowor) [inline docs/extensibility/30-details-summary.md:28]: Should this photo be displayed here?

File: docs/extensibility/30-details-summary.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ This table lists the available parameters of the **data.details.header**, **data
 
 Extra parameters might be available for specific widgets.
 
+#### Status Header
+
+The **data.details.status.header** accepts one widget (for example [Badge](./50-list-and-details-widgets.md#badge)), displayed on the right side of the **Status** section header. This widget is used to summarize the status of the resource.
+
+<img src="./assets/display-widgets/StatusBadge.png" alt="Example of a status badge widget" width="40%" style="border: 1px solid #D2D5D9">
```
Comment (OliwiaGowor) [inline docs/extensibility/30-details-summary.md:28]: Should this photo be displayed here or is it unnecessary?

File: docs/extensibility/30-details-summary.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ In the **data.details** section you can provide configuration of four optional c
 
 ### **header**, **status**, **body** and **health** Parameters
 
-This table lists the available parameters of the **data.details.header**, **data.details.status**, **data.details.health** and/or **data.details.body** section in your resource ConfigMap. You can learn whether each of the parameters is required and what purpose it serves. The **data.details.header**, **data.details.status**, **data.details.health** and **data.details.body** components are arrays of objects.
+This table lists the available parameters of the **data.details.header**, **data.details.status**, **data.details.health** and/or **data.details.body** section in your resource ConfigMap. You can learn whether each of the parameters is required and what purpose it serves. The **data.details.header**, **data.details.health** and **data.details.body** components are arrays of objects. The **data.details.status** component is an object that accepts **header** and **details** parameters. Within **data.details.status**, the **header** component is an object, and the **details** component is an array of objects.
```
Comment (grego952) [inline docs/extensibility/30-details-summary.md:11]:
```suggestion
This table lists the available parameters of the **data.details.header**, **data.details.status**, **data.details.health** and/or **data.details.body** section in your resource ConfigMap. You can learn whether each of the parameters is required and what purpose it serves. The **data.details.header**, **data.details.health** and **data.details.body** components are arrays of objects. The **data.details.status** component is an object that accepts the **header** and **details** parameters. Within **data.details.status**, the **header** component is an object, and the **details** component is an array of objects.
```

File: docs/extensibility/30-details-summary.md (hunk 2 above)
Comment (grego952) [reply docs/extensibility/30-details-summary.md:28]: I'd drop it. It's more difficult to maintain screenshots, plus it's well-described, so it doesn't need one.

**Review by grego952 — APPROVED** [review]

**Pull request merged**